contents/tutorials/prevent-fouc-ab-tests.md

```diff
@@ -0,0 +1,74 @@
+---
+title: How to prevent Flash of Unstyled Content (FOUC) during A/B tests
+date: 2024-11-08
```

Comment:
don't forget to update date when shipping

Reply:
```suggestion
date: 2024-11-13
```